### PR TITLE
Add NovHell plugin

### PR DESCRIPTION
**Pull request by elfing44**
Add NovHell

### CODE REVIEW

File: plugins/french/novhell.ts [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+        novelName = $(elem).find('figcaption a strong').first().text().trim();
+      }
+      const novelCover = $(elem).find('a img').attr('src');
+      const novelUrl = $(elem).find('a').attr('href');
+
+      if (novelUrl && novelName) {
+        novel = {
+          name: novelName,
+          cover: novelCover,
+          path: novelUrl,
```
Comment (nyagami) [inline plugins/french/novhell.ts:54]: novel path doesnt include the site domain
eg: `/novel/1` instead of `https://site.com/novel/1`

**Comment by nyagami** [comment]
Could you make the plugin icon squre?
Also please check this novel, some chapters have the same url
image

**Comment by elfing44** [comment]
> Could you make the plugin icon squre? Also please check this novel, some chapters have the same url image

The error comes from the site which has chapters which do not point to the correct link
I will correct the icon

**Comment by nyagami** [comment]
the icon is square, but too large, it should be around 10kb

**Review by nyagami — APPROVED** [review]

**Pull request merged**